daemon/mgr/cri.go

```diff
@@ -247,9 +249,41 @@ func (c *CriManager) Status(ctx context.Context, r *runtime.StatusRequest) (*run
 	return nil, fmt.Errorf("Status Not Implemented Yet")
 }
 
+func imageToRuntimeAPIImage(image *apitypes.ImageInfo) (*runtime.Image, error) {
```

Comment:
Could you add a comment for this function?
Actually I have no strong idea about RuntimeAPIImage.
Thanks. 😄